celery_config.py

```diff
@@ -94,6 +96,13 @@ def _beat_schedule():
                 "cron_task_generation_time_iso": BeatLazyFunc(get_utc_now_as_iso_format)
             },
         },
+        "regular_cleanup": {
+            "task": regular_cleanup_cron_task_name,
+            "schedule": crontab(minute="0", hour="2"),
```

Comment:
I'm curious how the 2 hour cron will fare. Don't anticipate it collapsing codecov, but I think this would be our most recurrent cron 👀

Reply:
oooooh that makes more sense 😂